draft-ietf-quic-manageability.md

```diff
@@ -181,6 +181,21 @@ used to separate QUIC packets. The length header field is variable length and
 its position in the header is also variable depending on the length of the
 source and destionation connection ID. See Section 4.6 of {{QUIC-TRANSPORT}}.
 
+## Use of Port Numbers
+
+Applications that have a mapping for TCP as well as QUIC are expected to
+use the same port number for both services. However, as with
+TCP-based services, especially when application layer information are
+encrypted, there are no guarantee that a specific application uses the
```

Comment:
"there **is** no guarantee" or "there are no guarantee**s**"; "specific application will use the"

Reply:
```suggestion
encrypted, there is no guarantee that a specific application will use the
```